stable/artifactory/templates/nginx-deployment.yaml

```diff
@@ -60,14 +66,23 @@ spec:
         volumeMounts:
         - name: nginx-volume
           mountPath: {{ .Values.nginx.persistence.mountPath | quote }}
+        {{- if .Values.nginx.tlsSecretName }}
+        - name: ssl-secret-volume
+          mountPath: "/var/opt/jfrog/nginx/ssl"
+        {{- end }}
         resources:
 {{ toYaml .Values.nginx.resources | indent 10 }}
       volumes:
       - name: nginx-volume
       {{- if .Values.nginx.persistence.enabled }}
         persistentVolumeClaim:
-          claimName: {{ template "artifactory.nginx.fullname" . }}
+          claimName: {{ .Values.artifactory.persistence.existingClaim | default (include "artifactory.nginx.fullname" .) }}
```

Comment:
Change `artifactory.persistence.existingClaim` to `nginx.persistence.existingClaim`

Reply:
Done